components/gfx/platform/windows/font_template.rs

```diff
@@ -60,9 +60,18 @@ impl FontTemplateData {
 
     pub fn native_font(&self) -> Option<NativeFontHandle> {
         if self.bytes.is_none() {
-            Some(descriptor_from_atom(&self.identifier))
-        } else {
-            None
+            let font = font_from_atom(&self.identifier);
+            let face = font.create_font_face();
+            let files = face.get_files();
+            if files.len() >= 1 {
+                if let Some(path) = files[0].get_font_file_path() {
```

Comment:
This could be written more clearly like:

```
if self.bytes.is_some() {
    return None;
}
let font = ...
let face = ...
let files = ...
let path = files.iter().next()?.get_font_file_path()?;
Some(NativeFontHandle { .. })
```